src/sentry/static/sentry/app/views/settings/organizationMembers/organizationRequestsView.tsx

```diff
@@ -0,0 +1,157 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {AccessRequest, Member} from 'app/types';
+import {addErrorMessage, addSuccessMessage} from 'app/actionCreators/indicator';
+import {Panel, PanelBody, PanelHeader} from 'app/components/panels';
+import {t, tct} from 'app/locale';
+import AsyncView from 'app/views/asyncView';
+import EmptyMessage from 'app/views/settings/components/emptyMessage';
+import routeTitleGen from 'app/utils/routeTitle';
+import SentryTypes from 'app/sentryTypes';
+
+import InviteRequestRow from './inviteRequestRow';
+import OrganizationAccessRequests from './organizationAccessRequests';
+
+type Props = AsyncView['props'] & {
+  requestList: AccessRequest[];
+  inviteRequests: Member[];
+  updateInviteRequests: (id: string) => void;
+  updateRequestList: (id: string) => void;
+  showInviteRequests: boolean;
+};
+
+type State = AsyncView['state'] & {
+  inviteRequestBusy: Map<string, boolean>;
```

Comment:
Instead of using a `Map` object (which is prob fine) you could also write this as `{[key: string]: boolean}`